app/views/docs/authentication-server.phtml

```diff
@@ -25,8 +25,8 @@
             <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Client, Account } from "appwrite";
 
 const client = new Client()
-    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
-    .setProject('5df5acd0d48c2');               // Your project ID
+    .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
+    .setProject('[PROJECT_ID]');               // Your project ID
```

Comment:
Based on this convention https://stackoverflow.com/questions/21503865/how-to-denote-that-a-command-line-argument-is-optional-when-printing-usage

Required arguments are denoted by angled brackets `<required_argument>` and optional ones as `[optional_argument]` . So I think we should follow this widely used convention as well .

Reply:
YES! but also this is why we need to move to markdown!!!!

We need to escape it every time because it's taken as an HTML tag!